This patch fixes a conflict between the bcopy macro defined via 
preprocessor directives and the one contained in string.h.

diff -x config.log -x config.status -ru work/termutils-2.0/tparam.c work/termutils-2.0/tparam.c
--- work/termutils-2.0/tparam.c	1995-09-03 01:53:43.000000000 +0200
+++ work/termutils-2.0/tparam.c	2013-03-18 00:06:38.159273382 +0100
@@ -21,7 +21,7 @@
 #endif
 
 #ifndef emacs
-#if defined(HAVE_STRING_H) || defined(STDC_HEADERS)
+#if !defined(HAVE_STRING_H) || !defined(STDC_HEADERS)
 #define bcopy(s, d, n) memcpy ((d), (s), (n))
 #endif
 
